src/downscaling_stats/analogs.f90

```diff
@@ -73,15 +73,17 @@ module subroutine find_analogs(analogs, match, input, n, threshold, weights, ski
         endif
 
         if (present(weights)) then
+            ! Set up the weights array as the inverse distance, allocating if necessary
             if (allocated(weights)) then
-                if (size(weights)/=size(analogs)) then
+                if ( size(weights) /= size(analogs)) then
```

Comment:
nit: extra space before size.

Reply:
thanks fixed. It's the little things that will keep this code a little cleaner